spark-connect/common/src/main/protobuf/spark/connect/commands.proto

```diff
@@ -1,19 +1,19 @@
 /*
-* Licensed to the Apache Software Foundation (ASF) under one or more
-* contributor license agreements.  See the NOTICE file distributed with
-* this work for additional information regarding copyright ownership.
-* The ASF licenses this file to You under the Apache License, Version 2.0
-* (the "License"); you may not use this file except in compliance with
-* the License.  You may obtain a copy of the License at
-*
-*    http://www.apache.org/licenses/LICENSE-2.0
-*
-* Unless required by applicable law or agreed to in writing, software
-* distributed under the License is distributed on an "AS IS" BASIS,
-* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-* See the License for the specific language governing permissions and
-* limitations under the License.
-*/
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
This file contains delta changes as well right?

Reply:
I mentioned in the description that they are the same, except for the `option java_package` part, we are `io.delta.connect.spark` while Spark Connect is `org.apache.spark.connect.proto`

Reply:
OK. What about then the rest proto files in the folder? Don't they need any adjustment?

Reply:
You are right 🤦 , thanks =))